app/models/carrier.rb

```diff
@@ -18,6 +18,19 @@ class Carrier < ApplicationRecord
 
   has_many_attached :photos
 
+  enum status: {
+    available: 0,
+    unavailable: 1,
+    disabled: 2,
+    sold: 4
+  }
+
+  alias_method :available_for_checkout?, :available?
+
+  def self.options_for_status_select
```

Comment:
This feels like responsibility of the rails helper since it is used only for the UI, wdyt?

Reply:
It might be that I was forgetting something about Rails forms. Since I'm using a hash enum if I just put the actual enum collection into the form helper than the values in the select are set to 0, 1, 2, 4 while AR is expecting the string values to come from the form. 

I'd be happy if there is a more idiomatic way to do this. (I've been working on this at night after the rest of the family is asleep that might have something to do with it 😉 )